### PR TITLE
add vsphere rhel

### PR DESCRIPTION
**Pull request by chaospuppy**
Adds a vsphere folder containing packer used to create a STIG'd, FIPS-enabled VM Template (VMWare parlance) and matching OVF (optional).

### CODE REVIEW

**Review by mjnagel — DISMISSED** [review]
From what I can check/test this LGTM - noting that I don't have a vSphere environment so haven't done an e2e test.

File: packer/scripts/rke2-config.sh [hunk 1]
```diff
@@ -20,6 +22,9 @@ chown -R root:root $config_dir/audit-policy.yaml
 mv -f $file_dir/default-pss.yaml $config_dir/default-pss.yaml
 chown -R root:root $config_dir/default-pss.yaml
 
+# Run restorecon to re-label files moved from /tmp
+restorecon -R $config_dir
+
```
Comment (mjnagel) [inline packer/scripts/rke2-config.sh:27]: This command `restorecon` is not available on Ubuntu it appears when reviewing the CI failure.

Comment (chaospuppy) [reply packer/scripts/rke2-config.sh]: Ah I should've caught that, thanks.  Now we're failing [here](https://github.com/defenseunicorns/uds-rke2-image-builder/actions/runs/9961465532/job/27523746300#step:6:1408), but this error seems to occur periodically while the airgap rke2 resources are being installed.  Thoughts?

Comment (mjnagel) [reply packer/scripts/rke2-config.sh]: Yeah I just kicked off a retry. That happens annoyingly often - I spent a while previously trying to use curl options, wget, tweaks to instance size, etc but couldn't isolate the source of the issue/a consistent fix. The airgap resource bundles are somewhat large so my best guess was network flakiness causing the curl to fail.

**Review by mjnagel — APPROVED** [review]

**Pull request merged**